### ISSUE TITLE
Scheduler crashes with 'Could not collect keys'

### ISSUE THREAD

**Issue by Bluemi** [curious-containers/curious-containers#22]
Job-id: 5d0209a96bf416114d981462 , Batch-ID: 5d0209a96bf416114d98146c

{'_id': '5d0209a96bf416114d98146c',
 'attempts': 0,
 'experimentId': '5d0209a96bf416114d981462',
 'history': [{'ccagent': None,
   'debugInfo': None,
   'node': None,
   'state': 'registered',
   'time': 1560414633.6023433},
  {'ccagent': None,
   'debugInfo': "Could not collect keys: ['25bde80f-7f3e-4c50-becc-643f3e3dd2ff']",
   'node': None,
   'state': 'failed',
   'time': 1560415150.562344}],
 'inputs': {'dvc_authentication_json': {'basename': 'dvc_authentication_json',
   'class': 'File',
   'connector': {'access': '2168e552-f7e2-4f4d-918e-77872cd600ba',
    'command': 'cc-connector-variable-to-file'},
   'dirname': '/tmp/red/inputs/402ed04d-346a-4457-a35a-3d4bc632356f',
   'nameext': '',
   'nameroot': 'dvc_authentication_json',
   'path': '/tmp/red/inputs/402ed04d-346a-4457-a35a-3d4bc632356f/dvc_authentication_json'},
  'dvc_file_to_execute': 'dvc/train_network.dvc',
  'dvc_path_to_working_repository': '/data/ldap/jonas/test_pcam',
  'dvc_servername': 'avocado01.f4.htw-berlin.de',
  'git_authentication_json': {'basename': 'git_authentication_json',
   'class': 'File',
   'connector': {'access': 'f5375c04-5c33-4308-a746-8cd449d6236e',
    'command': 'cc-connector-variable-to-file'},
   'dirname': '/tmp/red/inputs/54a46687-a1d7-4c80-9f5d-b265d7676e4f',
   'nameext': '',
   'nameroot': 'git_authentication_json',
   'path': '/tmp/red/inputs/54a46687-a1d7-4c80-9f5d-b265d7676e4f/git_authentication_json'},
  'git_name_of_tag': 'cc_0005_try_learning_rate_0.05',
  'git_path_to_working_repository': 'git.tools.f4.htw-berlin.de',
  'git_working_repository_name': 'test_pcam.git',
  'git_working_repository_owner': 'annusch'},
 'node': None,
 'notificationsSent': False,
 'outputs': {},
 'protectedKeysVoided': False,
 'registrationTime': 1560414633.6023433,
 'state': 'failed',
 'username': 'annusch'}

**Comment by Bluemi** [comment]
Probably fixed by [d459316](https://github.com/curious-containers/cc-agency/commit/d4593163e693501928fbc943ea79f5fefcdf26cb)
